### ISSUE TITLE
More than 40+ games available to mine cause fatal error

### ISSUE THREAD

**Issue by Steven-park** [DevilXD/TwitchDropsMiner#600]
So few days ago I decided to link more accounts on my twitch so the miner can get more drops.

Since then the program crashes almost immediately with the following error:

04:13:01: Fatal error encountered:
04:13:01: 
04:13:01: Traceback (most recent call last):
04:13:01:   File "main.py", line 155, in main
04:13:01:   File "twitch.py", line 587, in run
04:13:01:   File "twitch.py", line 747, in _run
04:13:01:   File "twitch.py", line 1657, in bulk_check_online
04:13:01:   File "asyncio\tasks.py", line 571, in _wait_for_one
04:13:01:   File "twitch.py", line 1387, in gql_request
04:13:01: exceptions.GQLException: [{'message': 'service error', 'path': ['channel', 'viewerDropCampaigns']}]
04:13:01: 
04:13:01: Exiting...
04:13:02: 
04:13:02: Application Terminated.
04:13:02: Close the window to exit the application.

So i waited a few days just in case it was a known issue but no updates and it must have been an error unique to me.

After some experimenting, I have found out that when I set the miner to mine everything on either the "Ending Soonest" or  "Low availability first", the crash occurs.

But when its on "priority only" the program doesnt crash immediately when no games are on the priority list.

So I individually went through each game and added it to the priority list to see which one was causing the issue.

Turns out when you add more than 40 games on the priority list, the miner becomes unstable and crashes.

I suspect no one else had this issue because normally prople just link the accounts they want for certain games and not link the one they dont need.

But I just wanted to mine everything possible so when the miner boots, the miner tries to pull all the games and as there are more than 40 games available to mine, the miner crashes because there are too many.

tl;dr: the miner has a bug when there are more than 40+ games available to mine, it crashes

**Comment by DevilXD** [comment]
Hello,

> i waited a few days just in case it was a known issue but no updates and it must have been an error unique to me.

Well, there already is an open issue for this: #590. Not sure how well you were checking, but with only 26 open issues and the bottom half of them being old and clearly about other things, checking the remaining half would take 5m at most.

The other information from this post is crucial though - the numer of games apparently influences the probability for the error to appear. That'd explain why I'm unable to reproduce this, as my list of games is about 8 entries long, with me adding the 9th to test the reports of others.

I'll need to close this one as a duplicate, but #590 will receive a comment containing the information you've found.
